skins/laika/src/components/pages/DonationForm.vue

```diff
@@ -39,5 +58,67 @@ export default Vue.extend( {
 		paymentTypes: Array as () => Array<String>,
 		addressCountries: Array as () => Array<String>,
 	},
+	data: function () {
+		return {
+			currentFormComponent: 'Payment',
+			buttonsVisibility: {
+				previous: false,
+				next: true,
+				submit: false,
+			},
+		};
+	},
+	computed: {
+		currentProperties: {
+			get(): object {
```

Comment:
Another instance where strong typing would both help (by making sure the types match for the component and inside the computed property) and be a pain.